index.html

```diff
@@ -3944,283 +3945,242 @@ <h4>updated</h4>
-                  <dd>
+<code><a>equivalentId</a></code> property except: a) it is associated with a
+single value rather than a ordered set, and b) the <a>DID</a> is defined to be
+the canonical ID for the <a>DID subject</a> within the scope of the containing
+<a>DID document</a>.
+              </p>
+              <p>
 The value of <code><a>canonicalId</a></code> MUST be a <a
 data-cite="INFRA#string">string</a> that conforms to the rules in Section <a
-href="#did-syntax"></a>.
-                  </dd>
-                  <dd>
-The relationship is a statement that the <code><a>canonicalId</a></code> value
-is logically equivalent to the <code>id</code> property value and that the
-<code><a>canonicalId</a></code> value is defined by the <a>DID Method</a> to be
-the canonical ID for the <a>DID subject</a> in the scope of the containing <a>DID
-document</a>.
-                  </dd>
-                  <dd>
-A <code><a>canonicalId</a></code> value MUST be produced by, and a form of, the
+href="#did-syntax"></a>. The relationship is a statement that the
+<code><a>canonicalId</a></code> value is logically equivalent to the
+<code>id</code> property value and that the <code><a>canonicalId</a></code>
+value is defined by the <a>DID Method</a> to be the canonical ID for the <a>DID
+subject</a> in the scope of the containing <a>DID document</a>. A
+<code><a>canonicalId</a></code> value MUST be produced by, and a form of, the
 same <a>DID Method</a> as the <code>id</code> property value. (e.g.
-<code>did:example:abc</code> == <code>did:example:ABC</code>)
-                  </dd>
-                  <dd>
-A conforming <a>DID Method</a> specification MUST guarantee that the
+<code>did:example:abc</code> == <code>did:example:ABC</code>). A conforming
+<a>DID Method</a> specification MUST guarantee that the
 <code><a>canonicalId</a></code> value is logically equivalent to the
 <code>id</code> property value.
-                  </dd>
-                  <dd>
+              </p>
+              <p>
 A requesting party is expected to use the <code><a>canonicalId</a></code> value
-as its primary ID value for the <a>DID subject</a> and treat all other equivalent
-values as secondary aliases. (e.g. update corresponding primary references in
-their systems to reflect the new canonical ID directive). <span
-class="issue">The testability of requesting parties is currently under debate and
-normative statements related to requesting parties may be downgraded in the
-future from a MUST to a SHOULD/MAY or similar language.</span>
-                  </dd>
-                </dl>
-
-                <div class="note" title="Equivalence and canonicalId">
-                  <p>
+as its primary ID value for the <a>DID subject</a> and treat all other
+equivalent values as secondary aliases (e.g. update corresponding primary
+references in their systems to reflect the new canonical ID directive).
+              </p>
+              <p>
```

Comment:
```suggestion
              <p class="note" title="Canonical equivalence">
```

And move to after the next paragraph